backend_storage/backend_storage.tf

```diff
@@ -1,10 +1,10 @@
 resource "aws_s3_bucket" "atul_state_bucket" {
-  bucket        = "atul--uw2-tf-bucket"
+  bucket        = "atul-tiwari-tf-state-bucket"
```

Comment:
I am very curious, why do you always change the name of the remote state bucket?